### ISSUE TITLE
Special HR transition tasks for v2 -> v3

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#4285]
Relates to https://github.com/uiowa/uiowa/issues/3752

**1/23/2025: If picking up this issue, please consult https://github.com/uiowa/uiowa/pull/8393#issuecomment-2610495261 for next steps.**

- [x] FAQs either need to transition to v2 accordions or FAQs content type needs to be accounted for in v3.
- [x] Content in Related Content field needs to be moved out so we can remove the field entirely. It is not currently rendered since the website moved from the old them to uids_base.
- [ ] There is a unit contact information block that is displayed in the sidebar if the unit is referenced. We either need to account for that as part of the p2lb conversion or move the content to another region.
- [ ] ???

**Note** As P2LB SiteNow Converter has been developed, there are several types of content in SiteNow v2 sites that do not have a 1:1 match in v3. It is hard to know to what extent HR is affected until the converter is turned on and the audit is run. Some pages might only be partially compatible and work might need to be done before or after conversion.

**Comment by joewhitsitt** [comment]
In the meeting today with Justin, they are working on FAQs and related content. Once they are done we could disable this functionality to prevent new items from being added.

Also, there are discussions about removing the unit contact block entirely and refocusing their content to be unitless on the visitor side. CTAs can be created as needed for pages that still require it. Otherwise, placing the information in another region could be possible (prefooter).

**Comment by briand44** [comment]
We should prevent them from adding FAQs. Should we be enabling the contact feature? Is there something we need to do with related content? Remove the field? A bug was identified in https://github.com/uiowa/uiowa/issues/8292

**Comment by joewhitsitt** [comment]
Also, what additional work will be needed to successfully run `sitenow_p2lb:cleanup`

Is this our first v2 custom to use p2lb? coronavirus was custom but we their conversion was different right?

**Comment by briand44** [comment]
Met with HR on 12/2 and Sean demoed P2LB. They are wanting to begin the P2LB transition in March, wrapping up end of April. They would like to get access to P2LB a couple of weeks in advance so they can put together some training resources for their distributed editors. Asked about the v3 stage site and they haven't been using that. That could be a good destination to turn on P2LB in advance when we approach their timeline. They plan to follow up with us on FAQ's and related content to let us know if we can do some cleanup by removing the content and functionality. They are still referencing contacts but are fine with those going away as part of the P2LB transition.

**Comment by pyrello** [comment]
> They are still referencing contacts but are fine with those going away as part of the P2LB transition.

It is my assumption that if we don't clean up this functionality ahead of time that when the P2LB transition happens these will just get ignored. We may want to test this out when we work on #8292

**Comment by joewhitsitt** [comment]
yeah, my concern with units/related content isn't the per page conversions but the final v2->v3 transition/clean-up with split config.

a test run in #8292 would bring any potential issues forward

**Comment by pyrello** [comment]
> yeah, my concern with units/related content isn't the per page conversions but the final v2->v3 transition/clean-up with split config.
> 
> a test run in #8292 would bring any potential issues forward

@joewhitsitt Are we trying to include removing the site split for HR in the p2lb process? I guess I'm not sure I'm understanding your concern.

**Comment by joewhitsitt** [comment]
No. My concern is that the site split is not clean/free of v2isms that will cause issue when the v2 split is removed.

Also since the unit reference field will no longer function by rendering a block (and the related content field since the uids_base transition), we could use the opportunity to clean up the site split.

**Comment by briand44** [comment]
From HR:

>Since yesterday, I removed the five instances of related content from hr.uiowa.edu.

**Comment by briand44** [comment]
Brian to check with Justin Friday afternoon 12/5, to see if we have approval to cleanup/remove faq's and related content from their v2 site.

**Comment by e-marie-w** [comment]
Grooming Convo: blocked until we get that approval. FAQ is the custom content type, also two fields on page content type that need to be dealt with before p2lb process. Custom view needs to be removed, along with config items related to the custom fields.

**Comment by briand44** [comment]
We have approval to move forward with FAQ and related content cleanup.

**Comment by briand44** [comment]
Next steps to finish this task: https://github.com/uiowa/uiowa/pull/8393#issuecomment-2610495261

**Comment by briand44** [comment]
Created a follow up issue for unit contact info https://github.com/uiowa/uiowa/issues/8457. I believe this issue can be closed now. Feel free to re-open if I am jumping the gun here.
